app/experimenter/static/rapid/components/forms/ExperimentForm.tsx

```diff
@@ -37,7 +38,10 @@ const ErrorList: React.FC<ErrorListProperties> = ({ errors }) => {
 };
 
 const ExperimentForm: React.FC = () => {
-  const formData = useExperimentState();
+  // eslint-disable-next-line
```

Comment:
What are you disabling here?

Reply:
Oh interesting, i'll check that out

Reply:
Ah, I see what's going on. I'd recommend changing this to

```ts
const formData = {...useExperimentState(), status: ExperimentStatus.DRAFT}
```

Reply:
+1. Also, it's usually more helpful to just ignore/disable a specific ESLint rule on the next line, if possible (vs shotgun disabling everything). But of course now that I've said that, I'll have to do some <kbd>git grep</kbd> to check our current eslint disable usages or the curiosity will drive me crazy.

Reply:
@k88hudson that doesn't work because then you're including `state` twice in the object and TS complains, once in the first and then explicitly in the second.

Reply:
I can't see an easy way to do this so I'm just gonna leave it how @tiftran wrote it.

Reply:
Oh it's htis:

```js
  const formData = {
    ...useExperimentState(),
    ...{ status: ExperimentStatus.DRAFT },
  };
```

Reply:
Really? `const formData = {...useExperimentState(), status: ExperimentStatus.DRAFT}` works for me (they should be equivalent?)